packages/trader/src/AppV2/Components/TradeParameters/Duration/duration.scss

```diff
@@ -19,15 +19,28 @@
     &__wheel-picker-container {
         padding: 0 var(--component-actionSheet-spacing-padding-md);
         min-height: 268px;
-        &__day {
-            min-height: 228px;
-        }
         &__loading {
             display: none;
         }
     }
+    &__days-input {
+        padding: 16px;
+        display: flex;
+        flex-direction: column;
+        gap: 16px;
+        &__expiry {
+            margin-top: 70px;
```

Comment:
--core-size-3500

Reply:
I forgot to change to quill token , let me do , thanks